src/sentry/static/sentry/app/components/search/sources/commandSource.jsx

```diff
@@ -25,6 +26,13 @@ const ACTIONS = [
       }),
   },
 
+  {
+    title: 'Toggle Translation Markers',
+    description: 'Toggles translation markers on or off in the application',
+    requiresSuperuser: true,
+    action: () => toggleLocaleDebug(),
```

Comment:
Can’t you just return the function

Reply:
Sure. The other one is like this tho, so I'll go back and get those after merging.